.github/workflows/call-backport-label-updater.yaml

```diff
@@ -0,0 +1,24 @@
+---
+    name: Call Backport Label Updater
+    on:
+      pull_request_target:
+        types:
+          - closed
+        branches:
+          - v1.15
+          - v1.14
+          - v1.13
+          - v1.12
+    jobs:
+      matrix:
+        branch: ["1.12", "1.13", "1.14", "1.15"]
```

Comment:
Just a heads up that the primary diff vs. the original submission is the use of this matrix to apply to various branches. I have not tested this at all, there could be basic syntax errors here. I'm not really sure how to validate or test github workflows like this. Treat this review with suspicion :)